src/Augurk.CommandLine/Properties/AssemblyInfo.cs

```diff
@@ -46,6 +46,6 @@
 //
 // You can specify all the values or you can default the Revision and Build Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("3.0.0")]
-[assembly: AssemblyFileVersion("3.0.0")]
-[assembly: AssemblyInformationalVersion("3.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.0.1")]
```

Comment:
Please undo the changes to this file (versioning is handled through GitVersion.yml)